code/game/machinery/computer/communications.dm

```diff
@@ -167,15 +167,19 @@
 					else if(!S.prerequisites_met())
 						to_chat(usr, "You have not met the requirements for purchasing this shuttle.")
 					else
-						if(SSshuttle.points >= S.credit_cost)
+						var/points_to_check
+						var/datum/bank_account/D = SSgoldmansachs.get_dep_account(ACCOUNT_CAR)
+						if(D)
+							points_to_check = D.account_balance
+						if(points_to_check >= S.credit_cost)
 							var/obj/machinery/shuttle_manipulator/M = locate() in GLOB.machines
 							if(M)
 								SSshuttle.shuttle_purchased = TRUE
 								M.unload_preview()
 								M.load_template(S)
 								M.existing_shuttle = SSshuttle.emergency
 								M.action_load(S)
-								SSshuttle.points -= S.credit_cost
+								D.adjust_money(-1 * S.credit_cost)

```

Comment:
why is this not just `D.adjust_money(-S.credit_cost)`

Reply:
i had troubles with BYOND throwing a shitfit about that

Reply:
there is literally no reason it should throw a shitfit. let it throw a shitfit so we can figure out why later when there's a weird runtime, because you should not be using multiply here.